crates/solver/src/solver.rs

```diff
@@ -118,7 +120,7 @@ pub struct Auction {
     pub run: u64,
 
     /// The GPv2 orders to match.
-    pub orders: Vec<LimitOrder>,
+    pub orders: Vec<Order>,
```

Comment:
This is a big change, but I support it (`LimitOrder` being both "liquidity" and "protocol order" was a bit messy IMO).

Reply:
I agree it's a big change. I give justification for it in the PR description.